README.md

```diff
@@ -0,0 +1,114 @@
+# Generate declarative jobsets for Hydra for a GitHub Project
+
+This tool uses Hydra's _declarative jobsets_ to automatically create Hydra jobsets for your GitHub project.
```

Comment:
Do we want to link to Hydra's docs on declarative jobsets as well? https://github.com/NixOS/hydra/blob/master/doc/manual/src/plugins/declarative-projects.md or some recent build on h.n.o?